CMakeLists.txt

```diff
@@ -664,14 +654,17 @@ if(JPSFIRE)
 endif()
 
 # todo make this system
-add_library(thirdParty STATIC ${THIRD_PARTY_SRC})
+add_library(thirdParty SHARED ${THIRD_PARTY_SRC})
 
 add_library(core STATIC ${source_files})
 if(CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
1st remark: cmake doc strongly suggest to not use `CMAKE_COMPILER_IS_GNUCXX` but instead use `${CMAKE_CXX_COMPILER_ID} EQUALS "GNU"` instead.

2nd remark clang and gcc link libs are now identical, you can say:
```cmake
if(${CMAKE_CXX_COMPILER_ID} MATCHES "^(GNU|Clang)$")
```